src/components/elements/input/PhoneInput.tsx

```diff
@@ -47,10 +47,10 @@ const TextMaskCustom = forwardRef<HTMLInputElement, CustomProps>(
   }
 );
 
-const PhoneInput = (props: TextInputProps) => {
+const PhoneInput = ({ InputProps, ...props }: TextInputProps) => {
   return (
     <TextInput
-      InputProps={{ inputComponent: TextMaskCustom as any }}
+      InputProps={{ inputComponent: TextMaskCustom as any, ...InputProps }}
       {...props}
```

Comment:
Mini scope-creep, but this feels like one where we should fix the input width. Similar to SSN, this is input accepts a fixed number of characters. (With some variability based on the presence of a country code).

```
      placeholder='(XXX) XXX - XXXX'
      inputWidth='155px'
```
What do you think of that?